collectors/feature/jira/src/main/java/com/capitalone/dashboard/client/team/TeamDataClientImpl.java

```diff
@@ -63,13 +60,6 @@ public int updateTeamInformation() {
 		if (CollectionUtils.isNotEmpty(teams)) {
 			updateMongoInfo(teams);
 			count += teams.size();
-		} else {
-			List<BasicProject> projects = jiraClient.getProjects();
-			
-			if (CollectionUtils.isNotEmpty(projects)) {
-				updateMongoInfoLegacy(projects);
-				count += projects.size();
-			}
```

Comment:
They do. In jira teams and projects are independent. A team can belong to multiple projects. A project can have multiple teams. Hence both inputs are needed as these are all valid filters:
- Project: Any, Team: myteam
- Project: myproject, Team: Any
- Project: myproject, Team: myteam
- Project: Any, Team: Any